custom/fitness-demo/README.md

```diff
@@ -0,0 +1,55 @@
+# Fitness Demo
+
+![Fitness Demo](./image.png)
+
+### Live example
+
+**[See it in action here ➡️](https://custom-fitness-demo.vercel.app)**
+
+---
+
+## What does this demo do?
+
+- Built on [NextJS](https://nextjs.org/)
+- Create a Daily instance using call object mode
```

Comment:
It would be nice to also include features from the class owners perspective (e.g. create private and public classes, mute attendee mics and cameras, etc. This list feels more targeted to the technical implementation perspective, than the feature set from the user's perspective, and I think both are important for understanding what you can do with this demo.